StatistikStadtZuerich/ld-data
# cube names

- Old config files
- Remove old documentation
- Move SPARQL to attic, most likely no longer used
- Rewrite readme
- Update copyright
- These two SPARQL queries are still used
- Add sparql directory
- Remove old scripts
- XRM update & fixes
- Update minor
- New file loader & XRM update
- Bootstrap from BLW pipeline
- WIP refactoring to latest pipeline
- No longer used
- Cleanup unused files
- Switch to ESM module loading
- ESM updates
- New XRM formating
- Adjust to headers of 2022
- Approxiation to the HDB mapping (outside XRM)
- Get as close as possible to the manuall CSVW mapping of HDB
- Adjust filters
- Add new mapping
- Update HDB mapping
- Update to latest CSV sources
- Refactor to 2022 CSV structure
- Adjust vocabularies
- Extend vocab
- Refactor CODELISTE
- Adjust KENNZAHL
- feat: added routing logic
- Fix dcterms IRI, closes #163
- Refactor DIMENSIONEN
- Refactor IRIs & vocab
- Update RAUM
- Update pipeline & filters
- Update mapping files
- fixed in map-step
- Small refactoring on HDB mapping
- Always ignore, needs some manual adjustments
- Latest node & new scripts
- Cleanup
- Unused steps
- Add fetch pipeline
- Basic fetch & extract
- Revert to original script
- Assets
- Build on all branches for now
- Does not seem to execute yet
- Add artifacts
- feat: added raum and zeit filter to the routing
- IRIs, not literals
- Adjust ZEIT mapping
- Disable or remove unused columns in mapping.
- feat: added routing to pipeline (#171)
- Irelevant according to SSZ, closing #170
- Not needed according to SSZ, closing #169
- Not required ATM according to SSZ, #169
- Not necessary ATM, issue #170
- Did not get cleaned up apparently
- No more conceptscheme
- Refactor for proper datatype
- Convert time to proper date literal
- Remove unused dimensions
- feat: added step to add cube names from routing csv
